stories/topics/proc-gcp-pull-backup-container-image.adoc

```diff
@@ -29,4 +29,9 @@ For EMEA regions (Europe, Middle East, Africa) run the following command instead
 ----
 $ export IMAGE={Registry}/ansible-on-clouds/ansible-on-clouds-ops-emea-rhel8:2.2.20230215
 $ docker pull $IMAGE --platform=linux/amd64
-----
\ No newline at end of file
+----
+
+[NOTE]
+====
+If your foundation deployment version is not {ImageRef}-00, refer to the tables on the link:https://access.redhat.com/documentation/en-us/ansible_on_clouds/2.x/html/ansible-automation-platform-from-gcp-release-notes/assembly-gcp-release-notes-24[Released versions] page for a matching deployment version, in the column *Ansible on Clouds version*, and find the corresponding operational image to use, in the column *Ansible-on-clouds-ops container image*, for the IMAGE environment variable.
```

Comment:
in the Ansible on Clouds version column.

Find the corresponding....

Reply:
fixed